src/inputs/input.css

```diff
@@ -0,0 +1,174 @@
+div.omrs-input-group {
+  margin-bottom: 1.5rem;
+  position: relative;
+  width: 20.4375rem;
+}
+
+input.omrs-input-underlined,
+input.omrs-input-filled,
+input.omrs-input-outlined {
+  font-size: 1rem;
+  display: block;
+  box-sizing: border-box;
+  width: 100%;
+  height: 3.5rem;
+  padding-left: 0.875rem;
+  outline: none;
+}
+
+input.omrs-input-underlined,
+input.omrs-input-filled {
+  border: none;
+  border-bottom: 2px solid var(--omrs-color-ink-lowest-contrast);
+  border-radius: 0.25rem 0.25rem 0px 0px;
```

Comment:
nit: use `0` instead of `0px`. The number is the only one where you don't need to provide a unit.

It makes no functional difference, but I'm just encouraging people to use `rem` whenever possible instead of px.